runtime/src/prioritization_fee.rs

```diff
@@ -125,6 +173,7 @@ impl PrioritizationFee {
 
                 self.metrics
                     .accumulate_total_prioritization_fee(transaction_fee);
+                self.metrics.update_prioritization_fee(transaction_fee);
```

Comment:
Is this called for vote transactions as well? Those will all be unprioritized. Should those be separated?

Reply:
Ah, good point! Perhaps better to filter out votes from updating fee cache all together. in https://github.com/solana-labs/solana/pull/29341